tests/conftest.py

```diff
@@ -311,6 +311,28 @@ def change_env_var(key, value):
             del os.environ[key]  # pragma: no cover
 
 
+_ENV_EMPTY = object()
+
+
+@pytest.fixture
+def override_env_var():
+
+    original_env = {}
+
+    def _environ_override(var, value):
+        original_env.setdefault(var, os.environ.get(var, _ENV_EMPTY))
+        os.environ[var] = value
+
+    try:
+        yield _environ_override
```

Comment:
any reason not just use the builtin ``monkeypatch.setenv``?

Reply:
No, I just didn't know about it!